mmv1/templates/terraform/examples/base_configs/test_file.go.erb

```diff
@@ -43,10 +43,11 @@ object.examples
     test_slug = "#{resource_name}_#{example.name.camelize(:lower)}Example"
 
     ignore_read = object.all_user_properties
```

Comment:
nit: Could we add a sort in a separate PR and then rebase this change to include it? That'll capture the true delta from your change- it's hard to see what has/hasn't changed as-is! That'll guard against spurious diffs during other similar changes to the set in the future.